action/uninstall.go

```diff
@@ -30,50 +34,126 @@ func Uninstall(chartName, home, namespace string) {
 	if err != nil {
 		log.Die("Failed to load chart: %s", err)
 	}
+	if err := deleteChart(c, namespace, true); err != nil {
+		log.Die("Failed to list charts: %s", err)
+	}
+	if !force && !promptConfirm("Uninstall the listed objects?") {
+		log.Info("Aborted uninstall")
+		return
+	}
 
 	log.Info("Running `kubectl delete` ...")
-	if err := deleteChart(c, namespace); err != nil {
+	if err := deleteChart(c, namespace, false); err != nil {
 		log.Die("Failed to completely delete chart: %s", err)
 	}
 	log.Info("Done")
 }
 
-func deleteChart(c *chart.Chart, ns string) error {
+// promptConfirm prompts a user to confirm (or deny) something.
+//
+// True is returned iff the prompt is confirmed.
+// Errors are reported to the log, and return false.
+//
+// Valid confirmations:
+// 	y, yes, true, t, aye-aye
+//
+// Valid denials:
+//	n, no, f, false
+//
+// Any other prompt response will return false, and issue a warning to the
+// user.
+func promptConfirm(msg string) bool {
+	oldState, err := terminal.MakeRaw(0)
+	if err != nil {
+		log.Err("Could not get terminal: %s", err)
+		return false
+	}
+	defer terminal.Restore(0, oldState)
+
+	f := readerWriter(log.Stdin, log.Stdout)
+	t := terminal.NewTerminal(f, msg+" (y/N) >> ")
```

Comment:
I found the chevrons `>>` unexpected:
```console:
$ helm uninstall alpine
pod/alpine
Uninstall the listed objects? (y/N) >> n
---> Aborted uninstall
```

Wouldn't it be more conventional to omit them and just leave a space after `(y/N)`? But I don't care that much about it and this PR LGTM otherwise.